pkg/cluster/nodeutils/util.go

```diff
@@ -44,34 +43,42 @@ func KubeVersion(n nodes.Node) (version string, err error) {
 
 // WriteFile writes content to dest on the node
 func WriteFile(n nodes.Node, dest, content string) error {
-	// create destination directory
-	err := n.Command("mkdir", "-p", path.Dir(dest)).Run()
+	// destination folder exists in tar and is created during cp from stdin
```

Comment:
Without knowing the hack above this snippet on its own doesn't make sense. It should reference that. But also probably we should not have multiple meanings for command. It is inconsistent with the other command implementations and breaks the potential for testing with a fake which is much of the point of the exec wrapper besides the node command abstraction utility built from it.